File: Algoritmos de ordenamiento/Java/SelectionSort.java
package Java;

public class SelectionSort {
    public static void selectionSort(int array[]) {
        int length = array.length;
        for (int i = 0; i < length - 1; i++) {
            int min_idx = i;
            for (int j = i + 1; j < length; j++)
                if (array[j] < array[min_idx])
                    min_idx = j;
            int temp = array[min_idx];
            array[min_idx] = array[i];
            array[i] = temp;
        }
    }
}

File: Algoritmos de ordenamiento/JavaScript/SelectionSort.js
function selectionSort(array) {
  let length = array.length;
  for (let i = 0; i < length; i++) {
    let min = i;
    for (let j = i + 1; j < length; j++) {
      if (array[j] < array[min]) {
        min = j;
      }
    }
    if (min != i) {
      let tmp = array[i];
      array[i] = array[min];
      array[min] = tmp;
    }
  }
  return array;
}

let array = [25, 84, 3, 2, 11];

console.log(selectionSort(array));